ps_checkout.php

```diff
@@ -81,18 +75,18 @@ class Ps_checkout extends PaymentModule
     ];
 
     public $configurationList = [
-        PayPalConfiguration::INTENT => Intent::CAPTURE,
-        PayPalConfiguration::PAYMENT_MODE => Mode::LIVE,
+        'PS_CHECKOUT_INTENT' => 'CAPTURE',
+        'PS_CHECKOUT_MODE' => 'LIVE',
```

Comment:
We should replace all the constants from new classes in the module constructor and install functions AND in the upgrade calls whereas we have an ClassNotFoundException raise when we upgrade the module for the first time.